apps/condo/domains/ticket/utils/unit.js

```diff
@@ -1,19 +1,21 @@
 const { get } = require('lodash')
 
+const parkingType = 'parking'
+
 function getSectionAndFloorByUnitName (property, unitName, unitType) {
     const res = {
         sectionName: null,
         sectionType: null,
         floorName: null,
     }
     if (unitName && unitType) {
-        const sections = get(property, ['map', `${unitType !== 'parking' ? 'sections' : 'parking'}`], [])
+        const sections = get(property, ['map', `${unitType !== parkingType ? 'sections' : parkingType}`], [])
         for (const section of sections) {
             for (const floor of section.floors) {
                 for (const unit of floor.units) {
                     if (unit.label === unitName) {
                         res.sectionName = section.name
-                        res.sectionType = section.type
+                        res.sectionType = unitType !== parkingType ? section.type : parkingType
```

Comment:
Look's a bit strange. May be better to do it more clear like this
```suggestion
                        res.sectionType = unitType !== PARKING_UNIT_TYPE ? SECTION_SECTION_TYPE : PARKING_SECTION_TYPE
```